.github/workflows/release-amd64.yaml

```diff
@@ -41,8 +41,10 @@ jobs:
           # We need to do such trick here as the format of the $GITHUB_REF 
           # is "refs/tags/<tag>"
           tag=$(echo $GITHUB_REF | cut -d/ -f3-)
-          tags=($tag)
-          tags+=($([[ "$tag" =~ "alpha"|"rc" ]] && echo "latest" || echo "stable"))
+          if [ "${tag}" = "main" ]; then
+              tag=$(./tools/packaging/release/release.sh next-release-version)
+          fi
+          tags=(${tag} "latest")
```

Comment:
Sorry for the dumb question, but is the case when we'd expect to trigger the release workflow_dispatch from a branch other than main just if are a re-creating an old release (e.g. after someone accidentally deleted it 😱). Under that circumstance do we still want the latest tag? I guess I'm trying to work out if there is a scenario that we have 3.n released and want to re-build 3.<n-1>?

Reply:
That's not a dumb question, it's a very interesting one.
Releases can be created from branches, and we won't have any branch that's used apart from main.  That was my rationale when adding this logic here.

Reply:
it should only occur if we're running from main, let me fix it.

Reply:
Updated, thanks!